src/services/user-group/services/courseScopeLists.js

```diff
@@ -23,17 +23,18 @@ module.exports = function setup() {
 		}
 		if (['true', 'all'].includes(substitution)) userQuery.$or.push({ substitutionIds: user._id });
 
+		const oneDayInMilliseconds = 864e5;
```

Comment:
```suggestion
		const oneDayInMilliseconds = 86400000;
```

Reply:
thats what I did at first too, but Codacy things that number is too long and might be inprecise on different systems, thus this form.

Reply:
what about `24 * 60 * 60 * 1000`? :-D

Reply:
or better:
```
const expiryDate = new Date(someDate);
expiryDate.setDate(expiryDate.getDate() + 1); // add one day
```